.github/workflows/run-tests.yml

```diff
@@ -91,7 +91,6 @@ jobs:
         PYTHONWARNINGS: default
       run: |
```

Comment:
wondering if it would help to add configuration `shell: bash`
and then to use `[[ -e ~/.local/share/venvs/inkscape/bin/activate ]] && \`